src/components/PayPalButtons.js

```diff
@@ -45,7 +45,7 @@ export default function PayPalButtons(props) {
         });
 
         return cleanup;
-    });
+    }, [isResolved, props.forceReRender]);
```

Comment:
We do have a couple other prop values that are primitives that should probably trigger a re-render. 
```suggestion
    }, [isResolved, props.forceReRender, props.fundingSource, props.shippingPreference]);
```